doc/source/conf.py

```diff
@@ -101,46 +101,16 @@
 
 variables = {
   "package_mroonga_version": os.environ["PACKAGE_MROONGA_VERSION"],
-  "package_mariadb101_version": "10.1.41",
-  "package_mariadb102_version": "10.2.27",
-  "package_mariadb103_version": "10.3.18",
-  "package_mariadb104_version": "10.4.8",
-  "download_base": "https://github.com/mroonga/mroonga/releases/download"
+  "download_base": "https://github.com/mroonga/mroonga/releases/tag"
 }
-variables["windows_mariadb101_package_link_label"] = \
-  "MariaDB {package_mariadb101_version} with Mroonga {package_mroonga_version}".format(**variables)
-variables["windows_mariadb101_package_link_basename"] = \
-  "{download_base}/v{package_mroonga_version}/mariadb-{package_mariadb101_version}-with-mroonga-{package_mroonga_version}".format(**variables)
-variables["windows_mariadb102_package_link_label"] = \
-  "MariaDB {package_mariadb102_version} with Mroonga {package_mroonga_version}".format(**variables)
-variables["windows_mariadb102_package_link_basename"] = \
-  "{download_base}/v{package_mroonga_version}/mariadb-{package_mariadb102_version}-with-mroonga-{package_mroonga_version}".format(**variables)
-variables["windows_mariadb103_package_link_label"] = \
-  "MariaDB {package_mariadb103_version} with Mroonga {package_mroonga_version}".format(**variables)
-variables["windows_mariadb103_package_link_basename"] = \
-  "{download_base}/v{package_mroonga_version}/mariadb-{package_mariadb103_version}-with-mroonga-{package_mroonga_version}".format(**variables)
-variables["windows_mariadb104_package_link_label"] = \
-  "MariaDB {package_mariadb104_version} with Mroonga {package_mroonga_version}".format(**variables)
-variables["windows_mariadb104_package_link_basename"] = \
-  "{download_base}/v{package_mroonga_version}/mariadb-{package_mariadb104_version}-with-mroonga-{package_mroonga_version}".format(**variables)
+variables["windows_mariadb_package_link_label"] = \
+  "MariaDB with Mroonga-{package_mroonga_version}".format(**variables)
+variables["windows_mariadb_package_link_basename"] = \
+  "{download_base}/v{package_mroonga_version}".format(**variables)
```

Comment:
We can expand `{download_base}` to `https://github.com/mroonga/mroonga/releases/tag` here because we have only one `download_base` use.

Reply:
I see.
I removed `download_base`.